galaxykit/namespaces.py

```diff
@@ -8,12 +7,11 @@ def create_namespace(client, name, group):
     except KeyError:
         groups = []
         if group:
-            group_id = groups.get_group_id(client, group)
```

Comment:
Not sure why this got removed - the use here is that if you pass the name of a group to this function, then that group gets given change_namespace and upload_to_namespace permissions.

Is this maybe no longer valid because of RBAC? But then I'd expect that there'd be additional changes that would support that, rather than just removing the group field.

Reply:
i'll second that question